Formula/s/semgrep.rb

```diff
@@ -211,7 +211,6 @@ def install
       # Install semgrep-core and spacegrep
       system "opam", "install", "--deps-only", "-y", "."
       system "opam", "exec", "--", "make", "core"
-      system "opam", "exec", "--", "make", "core-install"
```

Comment:
It's likely still needed for now as changes in https://github.com/returntocorp/semgrep/pull/8512 are not included in a release.

Reply:
yeah, upstream first, let's release on the semgrep side and fix along the way